projects/qt5.cmake

```diff
@@ -212,7 +212,7 @@ function(build_qt5)
     add_custom_target(qt5_build ALL
       COMMENT "Configuration complete...building qt5"
       WORKING_DIRECTORY ${QT5_REPO_PATH}
-      COMMAND make -j2
+      COMMAND make -j4
```

Comment:
At one point I just left it open ended with a -j ... used all my system resources and started paging...it was disasterous :)  I think i put it at -j2 so I could still do other things will it was compiling on my machine.  -j4 is fine.